src/Uno.UI/Microsoft/UI/Input/InputObject.cs

```diff
@@ -5,7 +5,7 @@ namespace Microsoft.UI.Input
 	public partial class InputObject
 	{
 		public Microsoft.UI.Dispatching.DispatcherQueue DispatcherQueue
-			=> Xaml.Window.Current.DispatcherQueue;
+			=> Microsoft.UI.Dispatching.DispatcherQueue.Main;
```

Comment:
Isn't this going to cause issues? (the main window could be closed)

Reply:
For now we have only a singleton `DispatcherQueue` (until we support multiple UI threads), so this will not be a problem. as `DispatcherQueue.Main` is matches the main UI thread, which stays the same for all windows.